tck/bridge-tck-main-portlet/src/main/java/com/liferay/faces/bridge/tck/tests/chapter_5/section_5_2/Tests.java

```diff
@@ -602,6 +602,41 @@ public PhaseId getPhaseId() {
 		return PhaseId.ANY_PHASE;
 	}
 
+	// Test is SingleRequest -- Render/Action
+	// Test #5.33 --
+	@BridgeTest(test = "headerPhaseListenerTest")
+	public String headerPhaseListenerTest(TestRunnerBean testRunner) {
```

Comment:
@stiemannkj1 Renaming this method from "renderPhaseListenerTest" to "headerPhaseListenerTest" is fine, but it causes some diffs across branches because the source formatter sorts the methods alphabetically. Would it be possible to Create a Portlet 3.0 branch class called HeaderTests.java that contains `headerPhaseListenerTest(TestRunnerBean)` and a backport Portlet 2.0 branch class that contains `renderPhaseListenerTest(TestRunnerBean)`?